azDeploySecureSub-sample.bicepparam

```diff
@@ -7,7 +7,7 @@ param namingConvention = '{workloadName}-{env}-{rtype}-{loc}-{seq}'
 param sequence = 1
 
 param identityObjectId = '<Valid Entra ID object ID for permissions assignment>'
-param vnetAddressSpace = '10.230.0.0/24'
-param redcapZipUrl ='<Valid Redcap Zip URL>'
-param redcapCommunityUsername  = '<Valid Redcap Community Username>'
+param vnetAddressSpace = '10.230.${sequence - 1}.0/24'
```

Comment:
Since this is the sample.bicepparam, I would add a comment to point out that vnetAddressSpace should be changed to something valid, as to not confuse what the `sequence` is meant for